webosose/ares-cli
# Add "/" to config category in js_service templates

:Release Notes:
Add "/" to config category in js_templates

:Detailed Notes:
In js_service type's templates, register "/config/setGreeting" method without "/".
It makes node framework's log printing.
`method category config should start with '/', adding one for you...`
Add "/" for "config/setGreeting" method in templates

:Testing Performed:
Pass unit test on ose target and auto target.
Pass eslint
Verify with CLI commands 
1. Generate web app and js service using ares-generate
2. Package them.
3. Install the created ipk to target. 
4. Connect target using ares-shell
5. Executes the installed js service by command, and check the log
    See PLAT-139166 for details.

:Issues Addressed:
[PLAT-139166] Add "/" to config category in js_templates
